# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: accessible/base/nsAccessibilityService.cpp
# Commit: a28e38db75a1
# Full Hash: a28e38db75a196c5fa776c255f8733783aa7614f
# Author: Morgan Reschenberg <mreschenberg@mozilla.com>
# Date: 2021-10-11 21:46:13
# Description:
#   Bug 1734541: Verify IPC doc exists before sending resolution update r=Jamie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D127860
# ==============================================================================

diff -r 0c98c980f2de -r a28e38db75a1 accessible/base/nsAccessibilityService.cpp
--- a/accessible/base/nsAccessibilityService.cpp	Mon Oct 11 15:11:07 2021 +0000
+++ b/accessible/base/nsAccessibilityService.cpp	Mon Oct 11 15:11:17 2021 +0000
@@ -402,7 +402,7 @@
     mozilla::PresShell* aPresShell, float aResolution) {
   if (StaticPrefs::accessibility_cache_enabled_AtStartup()) {
     DocAccessible* document = GetDocAccessible(aPresShell);
-    if (document) {
+    if (document && document->IPCDoc()) {
       nsTArray<mozilla::a11y::CacheData> data(1);
       RefPtr<AccAttributes> fields = new AccAttributes();
       fields->SetAttribute(nsGkAtoms::resolution, aResolution);
